setup/baseConfig.yaml

```diff
@@ -29,6 +29,7 @@ foreground: true|false <run blobfuse2 in foreground or background>
 # Common configurations
 read-only: true|false <mount in read only mode - used for Streaming and FUSE>
 allow-other: true|false <allow other users to access the mounted directory - used for FUSE and File Cache>
+nonempty: true|false <allow mounting on non-empty directory - used for FUSE>
```

Comment:
We should mention this somewhere in the readme or migration.md files

Reply:
this was not a seperate config in v1, it was a "-o' suboption only and even in v2 we plan to keep it that way instead of explicitly saying you put this string in config. For that reason, have not made any explicit mention of this in other places.